apps/tlon-mobile/src/components/GroupListItem.tsx

```diff
@@ -0,0 +1,45 @@
+import type { ListItemProps } from '@tloncorp/ui';
+import { ListItem } from '@tloncorp/ui';
+import { useCallback, useMemo } from 'react';
+
+import * as db from '../db';
+
+export const GroupListItem = ({
+  model,
+  onPress,
+  onLongPress,
+  ...props
+}: ListItemProps<db.Group>) => {
+  const handlePress = useCallback(() => {
+    onPress?.(model);
+  }, [onPress, model]);
+  const handleLongPress = useCallback(() => {
+    onLongPress?.(model);
+  }, [onLongPress, model]);
+  const unreadCount = useMemo(() => {
+    return db.getUnreadChannelCount(model);
+  }, [model]);
+  return (
+    <ListItem {...props} onPress={handlePress} onLongPress={handleLongPress}>
+      <ListItem.Icon
+        fallbackText={model.title?.[0]}
+        backgroundColor={model.iconImageColor}
+        imageUrl={model.iconImage}
+      />
+      <ListItem.MainContent>
+        <ListItem.Title>{model.title}</ListItem.Title>
+        {/* <ListItem.Subtitle>{model.latestPost?.text}</ListItem.Subtitle> */}
+      </ListItem.MainContent>
+      <ListItem.EndContent>
+        {model.lastPostAt ? (
+          <ListItem.Time>
+            {new Date(model.lastPostAt).toLocaleDateString()}
```

Comment:
This should be formatted date, and probably also be memoized -- not critical for this PR though.